lib/index.js

```diff
@@ -19,21 +19,26 @@ module.exports = {
 
     return {
       upload: async (file, customParams = {}) => {
-        const path = file.path ? `${file.path}/` : "";
+        const path = file.path ? `${file.path}/` : '';
         const formData = new FormData();
-
-        formData.append("files", file.buffer, {
-          filename: file.hash,
-          contentType: file.mime,
-        });
-        formData.append("files_names", file.hash);
-        formData.append("folder_paths", `${root}/${path}`);
+        const {
+          buffer,
+          hash,
+          mime,
+        } = file;
+
+        const options = {
```

Comment:
Can we add the options to the customParams so that users can specify if they want their images to be public or private?